website/projects/awssync.py

```diff
@@ -22,8 +26,37 @@ def button_pressed(self):
         :return: True if function executes successfully
         """
         self.logger.info("Pressed button")
+        self.logger.info(self.get_emails_with_teamids())
         return True
 
+    def get_all_mailing_lists(self):
+        """
+        Get all mailing lists from the database.
+
+        :return: List of mailing lists
+        """
+        mailing_lists = MailingList.objects.all()
+        mailing_list_names = [ml.email_address for ml in mailing_lists]
+        return mailing_list_names
+
+    def get_emails_with_teamids(self):
+        """
+        Create a tuple with email and corresponding teamID, where teamID is a concatenation of ID and semesterID.
```

Comment:
I can not deduce from anywhere why `teamid`, `teamID` or `projectID` (you mix these variable names in your implementation) is needed. Can you explain its purpose in the synchronization process? My best guess is that it is to distinguish between two or more projects with the same mailing list, which would be an edge case.